slack-plugin/src/main/kotlin/slack/gradle/SlackExtension.kt

```diff
@@ -38,9 +40,16 @@ import slack.gradle.dependencies.SlackDependencies
 @DslMarker public annotation class SlackExtensionMarker
 
 @SlackExtensionMarker
-public abstract class SlackExtension @Inject constructor(objects: ObjectFactory) {
-
-  internal val androidHandler = objects.newInstance<AndroidHandler>()
+public abstract class SlackExtension
+@Inject
+constructor(
+  objects: ObjectFactory,
+  globalSlackProperties: SlackProperties,
+  private val slackProperties: SlackProperties,
```

Comment:
For the uninitiated (me), where are there two instances of the type `SlackProperties`?

Reply:
One is global properties (I.e. root project) and the other is the local project. It's a bit of a holdover from before I built our new properties system though so I plan to consolidate them, as the new property API I have will gracefully fallback to the global one if the project doesn't specify